### ISSUE TITLE
Same expression twice causes error

### ISSUE THREAD

**Issue by willfaught** [motemen/gore#54]
```
$ gore -autoimport
gore version 0.2.5  :help for help
gore> time.Now().Add(time.Hour * 24).UnixNano() / 1e6
1446750220830
gore> time.Now().Add(time.Hour * 24).UnixNano() / 1e6
# command-line-arguments
/var/folders/s3/2h4dlfnj4qscnpjj7dy00mmr0000gn/T/709813128/gore_session.go:15: time.Now().Add(time.Hour * 24).UnixNano() / 1e+06 evaluated but not used
error: exit status 2
exit status 2
```

**Comment by vovanec** [comment]
+ 1

**Comment by itchyny** [comment]
This issue was resolved by #115 which fixed #75.
